build-tools/packages/build-cli/src/commands/layerCheck.ts

```diff
@@ -0,0 +1,82 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import path from "path";
+import { Flags } from '@oclif/core';
+import { LayerGraph, Timer, writeFileAsync } from "@fluidframework/build-tools";
+import { BaseCommand } from "../base";
+
+// eslint-disable-next-line @typescript-eslint/no-inferrable-types
+const packagesMdFileName: string = "PACKAGES.md";
+
+export class LayerCheck extends BaseCommand {
```

Comment:
Rename/move this file into `src/commands/check/layers.ts`. That will make the command `check layers`, and when we add policy-check, we'll make it `check policy`.

Similarly, when you work on genMonoRepoPackageJson, put it in `src/commands/generate/packageJson.ts`. Then we can add other things under the "generate" verb later.